app/models/subscription.rb

```diff
@@ -0,0 +1,76 @@
+class Subscription < ApplicationRecord
+  include Auditor::Model
+
+  enum status: %i[active trashed]
+  enum frequency: %i[daily]
+
+  has_many :alert_runs
+
+  validates :email, email_address: { presence: true }
+  validates :frequency, presence: true
+  validates :search_criteria, uniqueness: { scope: %i[email expires_on frequency] }
+
+  scope :ongoing, -> { active.where('expires_on >= current_date') }
```

Comment:
Likewise, I don't think this is used anymore

Reply:
I believe this is still being used here: https://github.com/dxw/teacher-vacancy-service/blob/feature/job-seeker-alerts/app/services/subscription_finder.rb#L10